bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/client/SerialTransportAdapter.java

```diff
@@ -39,12 +39,12 @@
  * The {@link SerialTransportAdapter} provides org.openhab.core.io.transport.serial
  * services to the Calimero library.
  * 
- * @ServiceProvider annotation (biz.aQute.bnd.annotation) automatically creates the file
```

Comment:
Is this change intended?

Reply:
Yes, it fixes the javadoc build. @ character in a javadoc was interpreted as javadoc tag.
This is only the documentation, the ServiceLoader tag is just below.